docs/README.md

```diff
@@ -0,0 +1,349 @@
+```
+
+<h3 id="createclient-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|`clientId`|body|string|true|A unique ID that will be used to reference the client once it has been created|
+|`sdkKey`|body|string|false|A unique key used to initialize the DVC SDK|
+|`options`|body|object|false|An object of launch parameters. See [DevCycle Docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/node#initialization-options) for a full list of options|
+|`enableCloudBucketing`|body|boolean|false|An option to use the cloud bucketing version of the SDK|
+
+> Example responses
+
+> Created
+
+```json
+{
+  "message": "Success"
+}
+```
+
+```json
+{
+  "message": "Missing environment key! Call initialize with a valid environment key"
+}
+```
+
+> 400 Response
+
+```json
+{
+  "error": "Invalid request: missing clientId"
+}
+```
+
+<h3 id="createclient-responses">Responses</h3>
+
+|Status|Meaning|Description|
+|---|---|---|
+|201|[Created](https://tools.ietf.org/html/rfc7231#section-6.3.2)|Created|
+|400|[Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)|Bad Request|
+
+<h3 id="createclient-responseschema">Response Schema</h3>
+
+Status Code **201**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» message`|string|true|A message indicating the result of initializing an SDK client|
+
+Status Code **400**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» error`|string|true|An error message indicating what went wrong with the request|
+
+### Response Headers
+
+|Status|Header|Type|Format|Description|
+|---|---|---|---|---|
+|201|Location|string||Location of the Client instance on the proxy server|
+
+<a id="opIdcommand"></a>
+
+`POST /client/{clientId}/command`
+
+*Invoke an sdk method*
+
+> Body parameter
+
+```json
+{
+  "command": "variable",
+  "isAsync": true,
+  "params": [
+    {
+      "location": "user/3"
+    },
+    {
+      "value": "string-key"
+    },
+    {
+      "value": "string-key-default"
+    }
+  ]
+}
+```
+
+<h3 id="command-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|`command`|body|string|false|The name of the SDK method to invoke|
+|`isAsync`|body|boolean|false|Tells the proxy weather to wait on the SDK method or not|
+|`params`|body|[anyOf]|false|Arguments to the  SDK method|
+|`» *anonymous*`|body|[LocationParam](#schemalocationparam)|false|An argument to an SDK method|
+|`»» location`|body|string|false|Should reference an instance on the proxy i.e. `user/3`|
+|`» *anonymous*`|body|[ValueParam](#schemavalueparam)|false|An argument to an SDK method|
+|`»» value`|body|string|false|none|
+|`clientId`|path|string|true|A unique ID of the SDK instance that the method is being invoked on|
+
+> Example responses
+
+> OK
+
+```json
+{
+  "entityType": "Variable",
+  "data": {
+    "_id": "638681f059f1b81cc9e6c7fa",
+    "key": "bool-var",
+    "type": "Boolean",
+    "value": true
+  },
+  "logs": []
+}
+```
+
+```json
```

Comment:
this should document the "errors from SDK" responses, like `exception` and `asyncError`